components/brave_wallet/browser/BUILD.gn

```diff
@@ -370,3 +375,7 @@ config("sardine_config") {
     "SARDINE_CLIENT_SECRET=\"$sardine_client_secret\"",
   ]
 }
+
+config("0x_config") {
+  defines = [ "ZERO_EX_API_KEY=\"$zero_ex_api_key\"" ]
+}
```

Comment:
Actually defines for API keys in gns are discouraged. Worth fixing here or making a separate issue. https://bravesoftware.slack.com/archives/C7VLGSR55/p1649750697403279